services/scan/src/store.ts

```diff
@@ -947,9 +947,10 @@ export class Store {
           : undefined
       );
 
+      // TODO: We should be waiting for the writeStream to drain before
+      // writing more data to it
       if (cvr) {
-        writeStream.write(JSON.stringify(cvr));
-        writeStream.write('\n');
+        writeStream.write(`${JSON.stringify(cvr)}\n`);
```

Comment:
I don't know if this does anything but it seems better to have fewer rather than more write events.